examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -82,13 +84,13 @@
         return CHIP_ERROR_INTERNAL;
     }
 
-    ReturnLogErrorOnFailure([nocSigner createOrLoadKeys:storage]);
+    ReturnLogErrorOnFailure([gNocSigner createOrLoadKeys:storage]);
 
-    ipk = [nocSigner getIPK];
+    ipk = [gNocSigner getIPK];
 
-    constexpr const char * identities[] = { kIdentityAlpha, kIdentityBeta, kIdentityGamma };
+    constexpr const char * identities[] = { kIdentityAlpha };
```

Comment:
Why this change?  What starts up the beta and gamma identities?

Reply:
Oups! Sorry I was locally playing with that in order emulate the behaviour of a single controller shutting down as it is going to be the most common behaviour (afaik).
Multiple controllers may triggers a few different crashes and that is not what I was trying to reproduce with this PR (Since the event loop is not stopped before the last controller shutdown, and if some timers / network events happens targeting a previous controller that have been shutdown it may crash...)
I will revert this change for this PR.